core/src/main/java/hudson/Functions.java

```diff
@@ -2289,4 +2290,49 @@ public static boolean isContextMenuVisible(Action a) {
             return true;
         }
     }
+
+    public static Icon tryGetIcon(String iconGuess, JellyContext context) {
+        if (iconGuess.startsWith("symbol-")) {
```

Comment:
In some code path `iconGuess` can be null, either needs a null check, or callers adjusted in jelly to prevent this

Noticed on the update center page when installing plugins: http://localhost:8080/updateCenter/